Controllers/HA4IoT.Controller.Main/Rooms/RoomConfiguration.cs

```diff
@@ -19,6 +20,7 @@ internal abstract class RoomConfiguration
 
         public IController Controller { get; }
         public CCToolsBoardController CCToolsBoardController { get; }
+        public KnxController knxController { get; }
```

Comment:
Please remove it here. This is my own controller setup and I don't own KNX devices. :-) You should create your own IoT project in another directory and reference the assemblies from this repository. My configuration (the "Main" controller) is intended to be used as an example and for controlling my home :-)